gulp/tasks/app.js

```diff
@@ -14,6 +15,7 @@ const vue = require('../plugins/vue');
 
 const jsPipe = lazypipe()
 	.pipe(() => wrap("if(errMgr.ok()) { %= body % }"))
+	.pipe(() => replace(/\bconst\b/g, 'var')) // 修正 Safari 的問題
```

Comment:
The proper way to solve this should be using strict mode instead (refer to https://stackoverflow.com/questions/67771875/).
However, Vue2 template compiler emits code containing `with`
(there're ways around this; for example by using vue-template-es2015-compiler.
But considering that this project will upgrade to Vue3 soon, doing so is pointless),
so I use the quick fix here.